testing/runner.ts

```diff
@@ -12,26 +12,24 @@ const DEFAULT_GLOBS = [
   "**/test.js"
 ];
 
-/* eslint-disable max-len */
 function showHelp(): void {
   console.log(`Deno test runner
 
 USAGE:
   deno -A https://deno.land/std/testing/runner.ts [OPTIONS] [FILES...]
 
 OPTIONS:
-  -q, --quiet               Don't show output from test cases 
-  -f, --failfast            Stop test suite on first error
-  -e, --exclude <FILES...>  List of file names to exclude from run. If this options is 
-                            used files to match must be specified after "--". 
-  
+  -q, --quiet               Don't show output from test cases
+  -f, --failfast            Stop running tests on first error
+  -e, --exclude <FILES...>  List of comma-separated files to exclude from run
+
 ARGS:
-  [FILES...]  List of file names to run. Defaults to: ${DEFAULT_GLOBS.join(
-    ","
-  )} 
-`);
+  [FILES...]  List of files to run.
+              A directory <dir> will expand to:
+                ${DEFAULT_GLOBS.map((s: string): string => `<dir>/${s}`).join(`
+                `)}
+              Defaults to "." when none are provided.`);
```

Comment:
s/"."/current directory

Reply:
Not a problem, but I was purposefully trying to be verbatim here i.e. `deno test` => `deno test .`.  For me "current directory" would take an extra second to parse. Are you sure?